src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -106,37 +106,37 @@ case class Failure(className: String, message: String)
 
 // KEEP ARGUMENTS SORTED BY NAME.
 // It simplifies lookup when checking if a parameter is here already.
-case class MLParams(
+class MLParams(
```

Comment:
Unfortunately, it seems this breaks conversions from `MLParams` to DataFrame columns, e.g. in `e.getCurrentResults()` (MLlib.scala line 80).